common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -78,10 +77,21 @@
 
         Sequence.prototype.updatePageTitle = function() {
             // update the page title to include the current section
-            var positionLink = this.link_for(this.position);
+            var positionLink = this.link_for(this.position),
+                currentSectionTitle;
```

Comment:
FYI we're down to just 55 `.coffee` files - less than 5% of our total frontend LOC!